src/Silk/Commands/Server/Config/ConfigCommands.Edit.Logging.cs

```diff
@@ -64,7 +63,7 @@ public partial class EditConfigCommands
                 return await _channels.CreateMessageAsync(_context.ChannelID, "`--channel`, `--webhook` or `--mobile true/false` is must be specified.");
             }
 
-            var modConfig = await _mediator.Send(new GetGuildModConfig.Request(_context.GuildID.Value));
+            var modConfig = await _mediator.Send(new GetGuildConfig.Request(_context.GuildID.Value));
```

Comment:
Don't need to change, I can take care of this after merge; Updating variable names cause modConfig no longer exists.